cli/js/runtime.ts

```diff
@@ -24,8 +22,8 @@ function getAsyncHandler(opName: string): (msg: Uint8Array) => void {
 // TODO(bartlomieju): temporary solution, must be fixed when moving
 // dispatches to separate crates
 export function initOps(): void {
-  OPS_CACHE = core.ops();
-  for (const [name, opId] of Object.entries(OPS_CACHE)) {
+  const opsMap = core.ops();
+  for (const [name, opId] of Object.entries(opsMap)) {
     core.setAsyncHandler(opId, getAsyncHandler(name));
   }
```

Comment:
Async handler registration is the last API bit that's a bit strange IMO and could be structured better